static_src/test/unit/util/cf_api.spec.js

```diff
@@ -79,6 +79,41 @@ describe('cfApi', function() {
     });
   });
 
+  describe('postCreateNewUserWithGuid()', function() {
+    it('test the request on the post request', function() {
+      const userGuid = 'fake-user-guid';
+      const expectedPayload = {
+        guid: userGuid,
+      };
+      const spy = sandbox.stub(http, 'post');
+      spy.returns(createPromise({ data: {}}));
+
+      cfApi.postCreateNewUserWithGuid(userGuid).then(() => {
+        const args = spy.getCall(0).args;
+        expect(spy).toHaveBeenCalledOnce();
+        expect(args[0]).toMatch('/users');
+        expect(args[1]).toEqual(expectedPayload);
+        done();
```

Comment:
If you use `done()` here you have to ad it to the `it` block on line 83. That way, it specifies to jasmine when it can move onto the next test.